tinyagentos/routes/system.py

```
@@ -119,6 +119,106 @@ async def _emit_fail(msg: str) -> None:
+        proc = await asyncio.create_subprocess_exec(
+            "systemctl",
+            *scope_flag,
+            "restart",
+            unit,
+            stdout=asyncio.subprocess.PIPE,
+            stderr=asyncio.subprocess.PIPE,
+        )
+        _, stderr = await asyncio.wait_for(proc.communicate(), timeout=45)
+    except asyncio.TimeoutError:
+        return {"unit": unit, "ok": False, "scope": scope_name, "detail": "restart timed out"}
+    except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)